### ISSUE TITLE
How to get camera-target and camera-orbit values for annotations

### ISSUE THREAD

**Issue by adesh02092000** [google/model-viewer#3296]
### How to get camera-target and camera-orbit values for annotations
I was trying to re-create this zoom-in effect, once the annotation is clicked. From the documentation, on my custom model
https://modelviewer.dev/examples/annotations/index.html#cameraViews

But how to get those perfect data-target and data-orbit attributes, set for my own annotations.

Is there a way to find those values using the editor?

**Comment by elalish** [comment]
Indeed, this is great feature of our editor! Simply go to the Hotspots section of the second tab and you can add them by clicking. When you download the scene, the HTML is all built for you, ready for further customization.
